_includes/tutorials/time-concepts/ksql/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,11 @@
+To process data based on *ingestion time*, also known as *log time*, which uses the Kafka record timestamp:
+
++++++
+<pre class="snippet"><code class="groovy">{% include_raw tutorials/time-concepts/ksql/code/tutorial-steps/dev/create-stream-logtime.sql  %}</code></pre>
++++++
+
+To process data based on *event time*, also known as *wall-clock time*, which uses a timestamp from within the record payload:
```

Comment:
```suggestion
To process data using a timestamp from a field within the record payload:
```

Reply:
@bbejeck I took the spirit of this suggestion and applied it in https://github.com/confluentinc/kafka-tutorials/pull/647/commits/623bcbee8ce6af37b033bc0ca999c64775fdd4b2. WDYT?